pyanaconda/display.py

```diff
@@ -154,7 +157,14 @@ def check_vnc_can_be_started(anaconda):
         vnc_startup_possible = False
 
     # disable VNC question if we were explicitly asked for text in kickstart
-    if anaconda.ksdata.displaymode.displayMode == DISPLAY_MODE_TEXT:
+    from pykickstart import constants as pykickstart_constants
+    display_mode_coversion_table = {
+        constants.DisplayModes.GUI: pykickstart_constants.DISPLAY_MODE_GRAPHICAL,
+        constants.DisplayModes.TUI: pykickstart_constants.DISPLAY_MODE_TEXT
+    }
+    display_mode = display_mode_coversion_table[anaconda.display_mode]
```

Comment:
You shouldn't be required to deal with this. Pykickstart should be just an implementation detail of the module.